velero.yaml

```diff
@@ -0,0 +1,88 @@
+package:
+  name: velero
+  version: 1.11.1
+  epoch: 0
+  description: Backup and migrate Kubernetes applications and their persistent volumes
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - wolfi-base
```

Comment:
```suggestion
```

we generally leave this out since it's implied during the image builds dependency on `wolfi-baselayout`